benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp_ompss/Makefile

```diff
@@ -0,0 +1,77 @@
+# Makefile for manual C++ version of PSyKAl'd NEMOLite2D.
```

Comment:
"...manual C++ OMPSS version of..." (or however OMPSS is capitalised).

Reply:
Done